dev/devicelab/bin/tasks/plugin_test.dart

```diff
@@ -16,5 +16,7 @@ Future<void> main() async {
         <String, String>{'ENABLE_ANDROID_EMBEDDING_V2': 'true'}),
     PluginTest('apk', <String>['-a', 'kotlin', '--platforms=android'], pluginCreateEnvironment:
         <String, String>{'ENABLE_ANDROID_EMBEDDING_V2': 'true'}),
+    // Test that Dart-only plugins are supported.
+    PluginTest('ios', <String>['--platforms=android'], dartOnlyPlugin: true),
```

Comment:
Copy-paste issue causing test failure.
```suggestion
    PluginTest('apk', <String>['--platforms=android'], dartOnlyPlugin: true),
```

```
Task result:
{
  "success": false,
  "reason": "podspec file missing at /opt/s/w/ir/x/t/flutter_devicelab_plugin_test.5yX12t/plugintest/ios/plugintest.podspec"
}
```

https://github.com/flutter/flutter/blob/f90b019c68edf4541a4c8273865a2b40c2c01eb3/dev/devicelab/lib/tasks/plugin_tests.dart#L129-L133

Reply:
🤦🏻